rts/Game/UI/MouseCursor.h

```diff
@@ -41,7 +41,12 @@ class CMouseCursor {
 	const std::string& GetName() const { return name; }
 	const HotSpot GetHotSpot() const { return hotSpot; }
 
-	float4 CalcFrameMatrixParams(const float3& winCoors, const float2& winScale) const;
+	float4 CalcFrameMatrixParams(const float3& winCoors, const float2& winScale) const
+	{
+		return CalcFrameMatrixParams(winCoors, winScale, false);
+	}
+
+	float4 CalcFrameMatrixParams(const float3& winCoors, const float2& winScale, const bool fullWindow) const;
```

Comment:
How about a default arg?
```suggestion
	float4 CalcFrameMatrixParams(const float3& winCoors, const float2& winScale, const bool fullWindow = false) const;
```

Reply:
Isn't that `fullWindow = false`? Or is there another way for default args in cpp?

Reply:
Ah, I'm stupid lol